### PR TITLE
Update: Docusaurus V3

### PR DESCRIPTION
**Pull request by VitroidFPV**
* Builds and runs the server fine on my machine, I checked a lot of the files for any formatting issues that may come up, and any new (following modern MDX standards) pages shouldn't be affected at all
  * Some of the archive pages may have incomplete code blocks that were made by indenting instead of fenced code blocks (` ``` ` on each end), but I ran them through a script that hopefully found most or all of them
  * Katex opening curly brackets will have to be escaped, as shown in the mdx example file
  * Other necessary changes listed in the [Docusaurus V3 Upgrade Guide](https://docusaurus.io/docs/migration/v3)

* The overall formatting is a little stricter, moving from MDX V1 to V3, but should lead to fewer unexpected errors if followed correctly

* You may need to clear `node_modules` and `package-lock.json` before running `npm i`. A clean-start clone should also work

### CODE REVIEW

File: docs/wiki/boards/MULTIFLITEPICO.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The features of the FC are detailed below:
 - MPU6050
 - Brushed motor support for 4 motors (all populated with fets)
 - M5 and M6 broken out (used when converting into a HEX)
-- 8A N Channel fets (10A <5secs)
+- 8A N Channel fets (10A \<5secs)
```
Comment (haslinghuis) [inline docs/wiki/boards/MULTIFLITEPICO.md:20]: Is `\` a escape character here to use `<` ?

Comment (VitroidFPV) [reply docs/wiki/boards/MULTIFLITEPICO.md:20]: yes, brackets need to be escaped https://docusaurus.io/docs/migration/v3#bad-usage-of--1

**Comment by haslinghuis** [comment]
Best to clone into new folder as even `git clean -xdf` does not always works.

**Review by haslinghuis — APPROVED** [review]
```
[WARNING] Docusaurus found 1 unused Markdown directives in file "docs/wiki/archive/GPS-Rescue-v4-5.md"
- :::CAUTION (191:3)
Your content might render in an unexpected way. Visit https://github.com/facebook/docusaurus/pull/9394 to find out why and how to fix it.
client (webpack 5.89.0) compiled successfully
```

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**